libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -20,14 +20,15 @@
  * Code-generating part of inline assembly.
  */
 
-#include <libsolidity/inlineasm/AsmCodeGen.h>
```

Comment:
Putting a corresponding header first detects missing dependencies. I think we should follow this style here: http://stackoverflow.com/a/614322/5197297 , i.e.

own header first, then own project headers, general project headers, boost, std.

This order detects errors that occur because some missing dependencies of a header are included only because they are included by all users of that header.